datafusion/sqllogictest/test_files/spark/README.md

```diff
@@ -21,6 +21,15 @@
 
 This directory contains test files for the `spark` test suite.
 
+## Implementation Status
+
+Implementing the `datafusion-spark` compatible functions project is still a work in progress.
+Many of the tests in this directory are commented out and are waiting for help with implementation.
```

Comment:
@shehabgamin On a separate note if we planning people to help contributing to this crate we probably need to attach a documentation or sample PR how to implement a function and test it with DF and Spark?